Modules/Loadable/Annotations/Widgets/qMRMLAnnotationROIWidget.cxx

```diff
@@ -229,3 +238,12 @@ void qMRMLAnnotationROIWidget::updateROI()
                            0.5*(bounds[5]-bounds[4]));
   d->ROINode->EndModify(wasModifying);
 }
+
+// --------------------------------------------------------------------------
+void qMRMLAnnotationROIWidget::onMRMLDisplayNodeModified()
+{
+  Q_D(qMRMLAnnotationROIWidget);
```

Comment:
Instead, you should add the check here. Similarly to what is done for `onMRMLNodeModified()` here: https://github.com/Punzo/Slicer/blob/69c78545742fe9dd258bafbe3fb44f008583c07c/Modules/Loadable/Annotations/Widgets/qMRMLAnnotationROIWidget.cxx#L134-L137